docs/moments.rst

```diff
@@ -71,5 +83,15 @@ astronomy), you can use::
     >>> sigma_map = cube.linewidth_sigma()  # doctest: +SKIP
     >>> fwhm_map = cube.linewidth_fwhm()  # doctest: +SKIP
 
+The ``linewdith_sigma`` computes a sigma linewidth map along the spectral axis, whereas
```

Comment:
typo.  Should also be `~spectral_cube.SpectralCube.linewidth_sigma` so that it properly links to the docs.  It would help to specify that sigma means standard deviation